### PR TITLE
Add confirm-add-suggested-token &  confirm-add-token components to Storybook

### PR DESCRIPTION
**Pull request by EtDu**
- Adds `confirm-add-suggested-token` & `confirm-add-token` components to Storybook with knobs to change the symbol and icon for the first token in the list.
- Both components are similar but have different token lists.

### CODE REVIEW

File: ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+export default {
+  title: 'Confirmation Screens',
+};
+
+const PageSet = ({ children }) => {
+  const symbol = text('symbol', 'DAI');
+  const image = text(
+    'Icon URL',
+    'https://assets.coingecko.com/coins/images/9956/small/dai-multi-collateral-mcd.png?1574218774',
```
Comment (ryanml) [inline ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js:17]: Should we save this to a static directory for the storybook to use? Then we aren't hot linking to the resource

Comment (EtDu) [reply ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js:17]: yeah good point

**Review by darkwing — DISMISSED** [review]

**Comment by Wrathcoin** [comment]
Merge

**Comment by kumavis** [comment]
adding images to the project?

**Review by kumavis — DISMISSED** [review]
adds redundant images

**Comment by EtDu** [comment]
Resolved

**Review by brad-decker — APPROVED** [review]

**Review by SirJosh1987 — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**